packages/components/src/SegmentedControl/src/SegmentedControlItem.module.css

```diff
@@ -0,0 +1,121 @@
+    --hop-SegmentedControlItem-display: flex;
+    --hop-SegmentedControlItem-align-items: center;
+    --hop-SegmentedControlItem-justify-content: center;
+
+    /* Small */
+    --hop-SegmentedControlItem-sm-padding: 0.125rem var(--hop-space-inset-md);
+
+    /* Medium */
+    --hop-SegmentedControlItem-md-padding: 0.375rem var(--hop-space-inset-md);
+
+    /* Slider */
+    --hop-SegmentedControlItem-slider-background: var(--hop-neutral-surface-selected);
+    --hop-SegmentedControlItem-slider-position: absolute;
+    --hop-SegmentedControlItem-slider-width: 100%;
+    --hop-SegmentedControlItem-slider-height: 100%;
+
+    /* Focused */
+    --hop-SegmentedControlItem-color-focused: var(--hop-neutral-text-hover);
+    --hop-SegmentedControlItem-background-focused: var(--hop-neutral-surface-hover);
+
+    /* Hovered */
+    --hop-SegmentedControlItem-color-hovered: var(--hop-neutral-text-hover);
+    --hop-SegmentedControlItem-background-hovered: var(--hop-neutral-surface-hover);
+
+    /* Pressed */
+    --hop-SegmentedControlItem-color-pressed: var(--hop-neutral-text-press);
+    --hop-SegmentedControlItem-background-pressed: var(--hop-neutral-surface-press);
+
+    /* Selected */
+    --hop-SegmentedControlItem-color-selected: var(--hop-neutral-text-selected);
+    --hop-SegmentedControlItem-background-selected: transparent;
+
+    /* Disabled */
+    --hop-SegmentedControlItem-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-SegmentedControlItem-background-disabled: var(--hop-neutral-surface);
+    --hop-SegmentedControlItem-cursor-disabled: auto;
+
+    /* Internal variables */
+    --color: var(--hop-SegmentedControlItem-color);
+    --cursor: var(--hop-SegmentedControlItem-cursor);
+    --background: var(--hop-SegmentedControlItem-background);
+    --padding: var(--hop-SegmentedControlItem-sm-padding);
+
+    cursor: var(--cursor);
+
+    position: var(--hop-SegmentedControlItem-position);
+
+    display: var(--hop-SegmentedControlItem-display);
+    gap: var(--hop-SegmentedControlItem-gap);
+    align-items: var(--hop-SegmentedControlItem-align-items);
+    justify-content: var(--hop-SegmentedControlItem-justify-content);
+
+    padding: var(--padding);
+
+    color: var(--color);
+
+    background: var(--background);
+}
+
+.hop-SegmentedControlItem, .hop-SegmentedControlItem__slider {
+    border: var(--hop-SegmentedControlItem-border);
+    border-radius: var(--hop-SegmentedControlItem-border-radius);
+}
+
+.hop-SegmentedControlItem__slider {
+    position: var(--hop-SegmentedControlItem-slider-position);
+    inline-size: var(--hop-SegmentedControlItem-slider-width);
+    block-size: var(--hop-SegmentedControlItem-slider-height);
+    background: var(--hop-SegmentedControlItem-slider-background);
+}
+
+.hop-SegmentedControlItem[data-hovered] {
+    --color: var(--hop-SegmentedControlItem-color-hovered);
+    --background: var(--hop-SegmentedControlItem-background-hovered);
+}
+
+.hop-SegmentedControlItem[data-focused] {
```

Comment:
data-focus-visible here instead.

focus = the outline shows on clicks + keyboard focus
focus-visible = the outline only show on keyboard focus